code/modules/mob/living/carbon/human/human.dm

```diff
@@ -188,14 +188,13 @@
 			stat("Chemicals", B.chemicals)
 
 		if(mind)
+			var/datum/antagonist/vampire/vamp = mind.has_antag_datum(/datum/antagonist/vampire)

```

Comment:
why don't you do:
```
for(var/a in mind.antag_datums)
    var/datum/antagonist/A = a
    A.handle_stat()
```

Something similarly-shaped for a `Life` handler would also be good